Sources/SourceKitLSP/Swift/OpenInterface.swift

```diff
@@ -81,4 +110,46 @@ extension SwiftLanguageServer {
       }
     }
   }
+
+  private func _findUSR(
+    request: LanguageServerProtocol.Request<LanguageServerProtocol.OpenInterfaceRequest>, 
+    uri: DocumentURI, 
+    snapshot: DocumentSnapshot,
+    symbol: String?,
+    completion: @escaping (Swift.Result<FindUSRInfo, SKDError>) -> Void
+  ) {
+    guard let symbol = symbol else {
+      return completion(.success(FindUSRInfo(position: nil)))
+    }
+    let keys = self.keys
+    let skreq = SKDRequestDictionary(sourcekitd: sourcekitd)
+
+    skreq[keys.request] = requests.find_usr
+    skreq[keys.sourcefile] = uri.pseudoPath
+    skreq[keys.usr] = symbol
+
+    if let compileCommand = self.commandsByFile[uri] {
+      skreq[keys.compilerargs] = compileCommand.compilerArgs
```

Comment:
Wasn't sure if this was necessary

Reply:
Compiler args are not used for the `find_usr` request, so there’s no need to specify them

https://github.com/apple/swift/blob/31032b9cae300878312dc21ddbd515c744553db6/tools/SourceKit/tools/sourcekitd/lib/Service/Requests.cpp#L3706-L3723